src/Meter.cpp

```diff
@@ -72,6 +73,7 @@ static const meter_details_t protocols[] = {
 #ifdef OMS_SUPPORT
 	METER_DETAIL(oms, OMS, "OMS (M-BUS) protocol based devices", 100, false), // todo what is the max. amount of reading according to spec?
 #endif
+	METER_DETAIL(modbus, Modbus, "Modbus meter", 64, true),
```

Comment:
Surround with ifdef fpor modbus support throughout the vode?

Reply:
It belongs to this todo to make the whole modbus stuff optional

- [ ] Add CMake module for libmodbus and make this feature optional

Still open ..